internal/command/deploy/machines_deploymachinesapp.go

```diff
@@ -384,3 +392,31 @@ func (md *machineDeployment) warnAboutProcessGroupChanges(ctx context.Context, d
 	}
 	fmt.Fprint(md.io.Out, "\n")
 }
+
+func (md *machineDeployment) doSanityChecks(ctx context.Context, lm machine.LeasableMachine, indexStr string) (err error) {
+	if md.skipSanityChecks {
+		return nil
+	}
+
+	if err = lm.WaitForSanityChecksToPass(ctx, indexStr); err == nil {
+		md.logClearLinesAbove(1)
+		return nil
+	}
+
+	time.Sleep(2 * time.Second) // Wait 2 secs to be sure logs have reached OpenSearch
```

Comment:
does it really needs 2 extra seconds when it already waited 10 secs for sanity checks?

Reply:
Good catch. It's not required (and was copy-pasted from `release_command` :)).